frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/comment/currentUserComment/currentUserComment.component.tsx

```diff
@@ -67,10 +66,10 @@ export const CurrentUserComment = ({
 	return (
 		<CommentWithButtonsContainer>
 			<CommentButtons>
-				<ErrorCommentButton onClick={() => onDelete(_id)}>
+				<ErrorCommentButton onClick={onDelete}>
```

Comment:
I think it would be more natural to do this:
```suggestion
				<ErrorCommentButton onClick={onDelete(comment)}>
```